Makefile.common

```diff
@@ -49,6 +49,5 @@ export PYTHON    ?= python
 #  SystemC paths are needed for verilator-driven builds
 export SYSTEMC_INCLUDE := $(BP_EXTERNAL_DIR)/include
```

Comment:
Does it work to move the SYSTEMC stuff to the Makefile.common also?

Reply:
I gotta check. Is the related problem is building the Verilator itself or using Verilator?

Reply:
Using verilator

Reply:
Looks fine either way actually.